source/transfer_list.rst

```diff
@@ -906,9 +906,26 @@ This manifest contains the SPMC attribute node consumed by the SPMD
 **Executable entry point information entry layout (XFERLIST_EXEC_EP_INFO)**
 
 This entry type holds an array of `entry_point_info_t` structures.
-`entry_point_info_t` is a data structure containing information used by a
-bootloader to initialize the execution state of an image. This entry provides
-the consumer with all information to execute subsequent images.
+`entry_point_info_t` is a TF-A-specific data structure [TF_BL31]_ used to
+represent the execution state of an image, that is, the state of general purpose
+registers, PC and SPSR.
+
+.. code:: c
+
+    typedef struct entry_point_info {
+        param_header_t h;
+        uintptr_t pc;
```

Comment:
uintptr_t can change size depending on the machine word size. Perhaps you need to use two different tags, depending on the architecture? Or use uint32_t here?

Reply:
As this architecturally defined i.e. 8 bytes wide on Aarch64, and 4 on Aarch32 makes most sense to define two separate tags.